contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -349,21 +353,19 @@ var BaseWorkspaceView = BaseView.extend({
 	},
 	edit_nodes:function(allow_edit, collection, is_clipboard, parent){
 		var UploaderViews = require("edit_channel/uploader/views");
-		$("#main-content-area").append("<div id='dialog'></div>");
```

Comment:
I feel like I might be missing where in the DOM the modals are mounted now? I see a lot of `el: ...` properties removed, but nothing to replace them?

Reply:
I've responded [here](https://github.com/learningequality/studio/pull/1077/files/c2bae7e31ee850395e9d4c983c9b8c1609876b5c#r248816681)